pkg/routing/redisrouter.go

```diff
@@ -321,21 +370,17 @@ func (r *RedisRouter) Stop() {
 	r.cancel()
 }
 
-func (r *RedisRouter) setParticipantRTCNode(participantKey livekit.ParticipantKey, participantKeyB62 livekit.ParticipantKey, nodeID string) error {
-	var err error
+func (r *RedisRouter) setParticipantRTCNode(participantKey livekit.ParticipantKey, participantKeyB62 livekit.ParticipantKey, nodeID string) (err error) {
 	if participantKey != "" {
-		err1 := r.rc.Set(r.ctx, participantRTCKey(participantKey), nodeID, participantMappingTTL).Err()
-		if err1 != nil {
-			err = errors.Wrap(err, "could not set rtc node")
-		}
+		err = r.rc.Set(r.ctx, participantRTCKey(participantKey), nodeID, participantMappingTTL).Err()
 	}
 	if participantKeyB62 != "" {
-		err2 := r.rc.Set(r.ctx, participantRTCKey(participantKeyB62), nodeID, participantMappingTTL).Err()
-		if err2 != nil {
-			err = errors.Wrap(err, "could not set rtc node")
-		}
+		err = r.rc.Set(r.ctx, participantRTCKey(participantKeyB62), nodeID, participantMappingTTL).Err()
```

Comment:
both keys could be available and this err could be `nil` wiping out the earlier error?

Reply:
oh you were intentionally wrapping both of them into the response. sure

Reply:
yeah that was the intention, but I clearly had typos in there and not really wrapping the results 😞